src/Symfony/Component/HttpKernel/EventListener/ExceptionListener.php

```diff
@@ -104,8 +106,34 @@ public static function getSubscribedEvents()
      *
      * @param \Exception $exception The \Exception instance
      * @param string     $message   The error message to log
+     *
+     * @deprecated since Symfony 4.4, use "logThrowable()" instead
      */
     protected function logException(\Exception $exception, $message)
+    {
+        @trigger_error(sprintf('The "%s::logException()" method is deprecated since Symfony 4.4, use "logThrowable()" instead.', __CLASS__), E_USER_DEPRECATED);
+
+        $this->doLogThrowable($exception, $message);
+    }
+
+    protected function logThrowable(\Throwable $exception, string $message): void
+    {
+        if (__CLASS__ !== \get_class($this) && __CLASS__ !== (new \ReflectionMethod($this, 'logException'))->getDeclaringClass()->getName()) {
```

Comment:
Doing the deprecation this way (same in `Application`) means one cannot have a BC/FC layer compatible with 3.4 up to 5.0 without triggering the notice.
Another way to do it would be to allow one to override both methods: if `logThrowable` is defined, all is fine - otherwise, if `logException` is overriden, then the deprecation happens.